src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -238,14 +238,14 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
                                         .options(this.zakenService.listCommunicatiekanalen()).build());
 
         this.editFormFields.set('behandelaar', new AutocompleteFormFieldBuilder().id('behandelaar').label('behandelaar')
-                                                                                 .value(this.zaak.behandelaar)
+                                                                                 .value(this.zaak.toekenning?.medewerker)
```

Comment:
Dit gaat nooit een waarde hebben toch? Die toekenning is bij de initialisatie van het scherm nooit gevuld die is voor de weg terug.

Reply:
Opgelost in nieuwe commit.